overlord/snapstate/backend/snapdata_test.go

```diff
@@ -49,6 +49,7 @@ func (s *snapdataSuite) TearDownTest(c *C) {
 }
 
 func (s *snapdataSuite) TestRemoveSnapData(c *C) {
+	dirs.SetSnapHomeDirs(filepath.Join(s.tempdir, "home"))
```

Comment:
this needs to undone, also it would make sense for all the tests to actually have a variant that set this with more than one entry

Reply:
Removing this line will cause the test to fail, due to `err = s.be.RemoveSnapData(info, nil)` which will after a few function calls, call `dirs.GetSnapHomeDirs()` inside `func DataHomeGlobs(opts *dirs.SnapDirOptions)`. 
Setting the directory being used by this test is required in order for the Globs to be properly retrieved. I don't mind working around it if you have any ideas on how to.

Reply:
+1 that we need a variant with more than one entry for homedirs